### DISCUSSION TITLE
Remove individual dependency from a group PR

### DISCUSSION THREAD

**Discussion by shishkin** [renovatebot/renovate#30203] [Suggest an Idea]
### Tell us more.

Sometimes a group PR fails to build due an issue with one of the dependencies. Currently I have to resort to manually updating the remaining dependencies. It is also impossible to close such PRs because grouped PRs are immortal. It would be great to have checkboxes for dependencies in a grouped PR to be able to remove individual dependency from a PR and retry the PR with the remaining dependencies.

**Comment by sebdanielsson** [comment]
Agree. Having the same problem with the angular monorepo group. Renovate proposes a zone.js version that is not compatible with Angular 18.2.1.

**Comment by HonkingGoose** [comment]
We have an issue about rolling up PRs into a single PR, I'm discussing how to deal with splitting of PRs from the rolled-up group:

- https://github.com/renovatebot/renovate/issues/4404#issuecomment-2306800683

**Comment by runspired** [comment]
I'd also like to see this as its pretty tedious to do tons of manual updates when one dep bump is bad
